app/scripts/lib/rpc-method-middleware/handlers/eth-accounts.ts

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
We should be getting this error on this line on the `yarn lint` CI run.
```
All imports in the declaration are only used as types. Use `import type`.eslint[@typescript-eslint/consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports)
```

```suggestion
import type {
```

Reply:
Looks like this rule isn't enabled in extension due to the eslint config packages being 3 years out of date. 

It would still be nice to make this a type import, but it isn't a required change.